pwa-backend/src/main/java/io/growingabit/mail/MailService.java

```diff
@@ -0,0 +1,48 @@
+package io.growingabit.mail;
+
+import java.io.UnsupportedEncodingException;
+
+import javax.mail.Message;
+import javax.mail.MessagingException;
+
+import org.apache.commons.codec.binary.Base64;
+
+import com.google.appengine.api.utils.SystemProperty;
+
+import io.growingabit.app.model.StudentEmailSignupStage;
+import io.growingabit.app.utils.Settings;
+
+public class MailService {
+
+  public static Message sendVerificationEmail(final StudentEmailSignupStage studentEmailSignupStage) throws UnsupportedEncodingException, MessagingException {
+
+    String verificationLink = createVerificationLink(studentEmailSignupStage);
+
+    String subject = "[Growbit] Verifiy your email";
+    String htmlBody = "<p>Welcome to Growbit!</p>";
+    htmlBody += "<p>Please verify your email address by clicking this link:</p>";
+    htmlBody += "<p><a href='" + verificationLink + "'></p>";
+    htmlBody += "<p>This link will expire in seven days</p>";
+    htmlBody += "<p></p>";
+    htmlBody += "<p><i>Growbit team</i></p>";
+
+    MailObject mailObject = new MailObject.Builder(studentEmailSignupStage.getData().getEmail(), subject).addBcc(Settings.getConfig().getString("io.growingabit.mail.bcc")).withHtmlBody(htmlBody).build();
+
+    return MailGaeApi.sendEmail(mailObject);
+  }
+
+  private static String createVerificationLink(final StudentEmailSignupStage studentEmailSignupStage) throws UnsupportedEncodingException {
+    String verificationLink = "";
+    if (SystemProperty.environment.value() == SystemProperty.Environment.Value.Production) {
+      verificationLink += "https://" + SystemProperty.applicationId.get() + ".appspot.com";
+    } else {
+      verificationLink += "http://localhost:8888";
+    }
+    return verificationLink + createVerificationCode(studentEmailSignupStage);
```

Comment:
Per ora ok, poi ci dobbiamo pensare su questa cosa. Immagino metteremo un custom domain dopo...

Reply:
ok

Reply:
La porta l'ho dovuta impostare via maven alla 8080, occhio. Quindi quel pezzo non fa! Cambia con la 8080, poi pensiamo come farlo in modo furbo